pandas/tests/plotting/test_misc.py

```diff
@@ -80,12 +80,22 @@ def test_autocorrelation_plot(self):
         ax = autocorrelation_plot(self.ts, label="Test")
         self._check_legend_labels(ax, labels=["Test"])
 
+        s = Series([1, 2, 3, 4, 5])
+        ax = autocorrelation_plot(s)
+        assert len(ax.get_lines()[-1].get_xdata()) == len(s)
+
     @pytest.mark.slow
     def test_lag_plot(self):
         from pandas.plotting import lag_plot
 
         _check_plot_works(lag_plot, series=self.ts)
         _check_plot_works(lag_plot, series=self.ts, lag=5)
+        _check_plot_works(lag_plot, series=self.ts, figsize=(15, 8))
+
+        lag = 2
+        s = Series([1, 2, 3, 4, 5])
+        ax = lag_plot(s, lag=lag)
+        assert len(ax.collections[0].get_offsets()) == (len(s) - lag)
```

Comment:
Thanks for your input @charlesdong1991 . I think you make a good point about `figsize` maybe not being a parameter we should have in `lag_plot` and `autocorrelation_plot`.  But an argument could be made to the usefulness of being able to just `autocorrelation_plot(..., figsize=(x, y))` instead of having to `fig, ax = plt.subplots(figsize=(x, y))` before. Especially to avoid having to import matplotlib.

Going to try to test the `figsize`.